src/app/containers/StoryPromo/index.jsx

```diff
@@ -132,15 +132,20 @@ const StoryPromoContainer = ({
 
   const useLargeImages = promoType === 'top' || promoType === 'leading';
 
+  const headingTagOverride = promoType === 'recommendation' ? 'div' : null;
```

Comment:
I'm not sure that overriding the promoType with that of 'recommendation' is the best way to go about this - since promoType is linked to the layout of the promo. See the screenshots above. 

Could you please share an example page on localhost where this applies, so I can see this in context?

Reply:
Thanks yes that makes sense. I was thinking that the Recommendations onward journey container https://github.com/bbc/simorgh/issues/6266 which hasn't been built yet would have a new promo type. I didn't realise the promoTypes were based just on layout - thanks for the screenshots 👍

Reply:
@sareh There is a [paper doc](https://paper.dropbox.com/doc/WS-CPS-STYs-Recommendations-OJ--Ay8L3UWVlM~fjzUfmlSQqGY_Ag-MYZwLip814C7EgKWczhvM#:uid=936854692642424538027257&h2=Front-end) that might help with the context. Have you come across it.

Reply:
@sadickisaac Thank you for sharing that document. No I haven't come across the requirements for recommendations.